docs/general/events.md

```diff
@@ -17,74 +17,56 @@ network connection severed, etc.
+* Events MAY have [extended](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.44.0/specification/logs/data-model.md#field-attributes) attributes
   attributes that provide additional context about the event.
-* It MAY contain a _payload_ (body) that describes the specific details of the
-  named event.
-* The event name uniquely identifies event structure / type of the _payload_ (body)
-  and the set of attributes.
-* The _payload_ (body) MAY contain any type supported by the OpenTelemetry data
-  model for the log [body](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#field-body)
-  and the semantic conventions will define the expected structure of the _payload_
-  (body) for the event.
-* The _payload_ (body) SHOULD be used to represent the structure of the event.
-
-Recommendations for defining events:
-
-* Use the _payload_ (body) to represent the details of the event instead of a
-  collection of [standard](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.44.0/specification/common#attribute)
-  attributes.
-* Events SHOULD be generated / produced / recorded using the
-    [Logs API](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.44.0/specification/logs/api.md#emit-a-logrecord)
-    to ensure that the event is created using the configured SDK instance.
-  * The Emit Event API is not yet available in all OpenTelemetry SDKs. Check [spec-compliance matrix](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.44.0/spec-compliance-matrix.md#logs) to see the implementation status in corresponding language.
-* It's NOT RECOMMENDED to prefix the _payload_ (body) _fields_ with the `EventName` to
-  avoid redundancy and to keep the event definition clean.
-* The events SHOULD document their semantic conventions including event name,
-  attributes, and the payload.
+* Events MAY contain a body.
+* The event name uniquely identifies event structure: the type of the body and the
+  set of attributes.
+* The body MAY contain any type supported by the OpenTelemetry data
+  model for the log [body](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.44.0/specification/logs/data-model.md#field-body).
 
 Recommendations on using attributes vs. body fields:
 
-* If the field should be comparable across events with different `EventName` (or between an event and other telemetry items),
-  it should be an attribute.
-* If the field is specific to the event itself, then it should be a body field.
-* Body fields that belong to events with different event names are not comparable.
+It's RECOMMENDED to use the collection of attributes to represent the details of
+the event.
+
+Body SHOULD be used when bridging logs and MAY be used when mapping externally defined
```

Comment:
can we say something about bridging log attributes (e.g. https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/13787) to event attributes, and that this guidance only applies to something like log "message"?

Reply:
I suggest removing this paragraph.
I think we should have a separate document on "log bridge" guidelines and semantic conventions.

Reply:
discussed at the Logs SIG meeting, let's remove it for now and revisit if we have use-cases for body in semconv

Reply:
removed body and guidelines around it.